lib/aurora_bootstrapper/exporter.rb

```diff
@@ -4,30 +4,43 @@ module AuroraBootstrapper
   class Exporter
     attr_reader :client
 
-    def initialize( client:, prefix: "", export_bucket:, blacklisted_tables: "", whitelisted_tables: "", blacklisted_fields: "" )
+    def initialize( client:, prefix: "", export_bucket:, blacklisted_tables: "", whitelisted_tables: "", blacklisted_fields: "", notifier: nil )
       @match              = "#{prefix}.*"
       @export_bucket      = export_bucket
       @blacklisted_tables = blacklisted_tables.split(",")
       @whitelisted_tables = whitelisted_tables.split(",")
       @blacklisted_fields = blacklisted_fields.split(",")
       @client             = client
+      @notifier           = notifier
+      d                   = DateTime.now
+      d_str               = d.strftime("%Y-%m-%d")
+      @export_date        = ENV.fetch( 'EXPORT_DATE', d_str )
```

Comment:
since this is only used in Table and comes from an env var, what's the thinking behind passing this through Exporter and Database?

Reply:
Notifier also needs this info

Reply:
I created @export_date in exporter, so that the data files and state file will be under the same datetime subfolder even if the job may run longer than one day